test/Mapping/Formatter/Exception/InvalidValueTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types = 1);
+
+namespace DASPRiD\FormidableTest\Mapping\Formatter\Exception;
+
+use DASPRiD\Formidable\Data;
+use DASPRiD\Formidable\Mapping\Formatter\BooleanFormatter;
+use DASPRiD\Formidable\Mapping\Formatter\Exception\InvalidValue;
+use PHPUnit_Framework_TestCase as TestCase;
+
+/**
+ * @covers DASPRiD\Formidable\Mapping\Formatter\Exception\InvalidValue
+ */
+class InvalidValueTest extends TestCase
+{
+    public function testMessageFromNonBoolean()
+    {
+        $this->assertSame(
```

Comment:
Changes all assertions that were `==` too `===` (e.g. changed all `assertEquals` to `assertSame`)